tests/c/keep_includes.c

```diff
@@ -0,0 +1,18 @@
+//===--- keep_includes.c - test input file for iwyu -----------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+#include "keep_b.h"
```

Comment:
Can you add an explanatory comment before here that explains the feature being tested? It looks entirely pointless in isolation :)

Something like:
```
...
//===----------------------------------------------------------------------===//

// Test that the --keep command-line argument forces us to keep an unused
// (and empty) header.

#include "keep_b.h"
...
```